Wrappers/Python/ccpi/web_viewer/README.md

```diff
@@ -5,4 +5,6 @@ Follow these steps:
 - Create the conda environment:
     - `mamba env create -f dev-environment.yml`
 - With the activated conda environment you can now start the web application:
-    - `python web_app.py path/to/folder/of/data/to/use`
\ No newline at end of file
+    - `python web_app.py path/to/folder/of/data/to/use`
+- If you want to use the 2D viewer pass the 2D arg to the script i.e. --2D or -d:
```

Comment:
you could add info on your entry point here

Reply:
I found --2D had to go before the path to the data, please can you make that clear here